<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>960</width>
        <height>549</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,1,0,0" columnstretch="0,1">
       <item row="3" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Подписи</string>
         </property>
         <layout class="QGridLayout" name="labelGridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="showLineouts">
            <property name="text">
             <string>Сноски</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="deleteLabelAttributePushButton">
            <property name="text">
             <string>Удалить атрибут</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="addLabelAttributePushButton">
            <property name="text">
             <string>Добавить атрибут</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Размер подписи</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="labelSizeEdit">
            <property name="suffix">
             <string> мм</string>
            </property>
            <property name="singleStep">
             <double>0.200000000000000</double>
            </property>
            <property name="value">
             <double>7.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <widget class="QTableView" name="labelAttributeTableView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Атрибуты</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QPushButton" name="deleteAttributePushButton">
            <property name="text">
             <string>Удалить атрибут</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="addAttributePushButton">
            <property name="text">
             <string>Добавить атрибут</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QTableView" name="attributeTableView">
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="4">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Диаграммы</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Заголовок</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="mDeleteDiagramm">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="mImportFromLayer">
            <property name="text">
             <string>Взять из слоя...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="mAddDiagramm">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="mSaveToFile">
            <property name="text">
             <string>В файл...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="mReadFromFile">
            <property name="text">
             <string>Из файла...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="5">
           <widget class="QListWidget" name="mDiagrammsListWidget"/>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QLineEdit" name="titleEdit"/>
          </item>
          <item row="4" column="0" colspan="5">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Размер круга диаграммы</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QRadioButton" name="fixedSizeRadioButton">
               <property name="text">
                <string>Фиксированный</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="fixedDiagrammSize">
               <property name="suffix">
                <string> мм</string>
               </property>
               <property name="maximum">
                <double>999999.000000000000000</double>
               </property>
               <property name="value">
                <double>15.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="scaledSizeRadioButton">
               <property name="text">
                <string>Масштабируемый</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QFrame" name="scaledSizeFrame">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="scaledSizeGridLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Максимальный диаметр</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Атрибут</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Минимальный диаметр</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1" colspan="2">
                 <widget class="QDoubleSpinBox" name="minDiagrammSize">
                  <property name="suffix">
                   <string> мм</string>
                  </property>
                  <property name="decimals">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <double>999999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>3.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1" colspan="2">
                 <widget class="QDoubleSpinBox" name="maxDiagrammSize">
                  <property name="suffix">
                   <string> мм</string>
                  </property>
                  <property name="decimals">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <double>999999999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>15.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Макс. значение</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDoubleSpinBox" name="scaleEdit">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <double>999999999.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>300000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QPushButton" name="scalePushButton">
                  <property name="text">
                   <string>Вычислить</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="3">
                 <layout class="QHBoxLayout" name="horizontalLayout"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
